src/current/cockroachcloud/export-logs.md

```diff
@@ -377,6 +379,117 @@ Perform the following steps to enable log export from your CockroachDB {{ site.d
+
+        1. Consult the log export entry on the [CockroachDB {{ site.data.products.cloud }} API Reference](https://www.cockroachlabs.com/docs/api/cloud/v1.html#post-/api/v1/clusters/-cluster_id-/logexport) and select the **Schema** tab to view the supported log configuration options, and determine the customized logging configuration you would like to use.
+
+            For example, consider the following configuration:
+
+            {% include_cached copy-clipboard.html %}
+            ~~~json
+            {
+             "type": "AZURE_LOG_ANALYTICS",
+             "log_name": "default",
+             "auth_principal": "{workspace_id}",
+             "azure_shared_key": "{primary_or_secondary_key}",
+             "redact": true,
+             "region": "",
+             "omitted_channels": [ "SESSIONS", "SQL_PERF"],
+             "groups": [
+                     {
+                         "log_name": "sql",
+                         "channels": ["SQL_SCHEMA", "SQL_EXEC"],
+                         "redact": false
+                     },
+                     {
+                         "log_name": "devops",
+                         "channels": ["OPS", "HEALTH", "STORAGE"],
+                         "min_level": "WARNING"
+                     }
+             ]
+            }
+            ~~~
+
+            This configuration:
+            - Enables [redaction](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/configure-logs#redact-logs) globally for all log entries emitted to Azure Monitor.
+            - Does not send log entries in the `SESSIONS` and `SQL_PERF` logging channels.
+            - Sends log entries in the `SQL_SCHEMA` and `SQL_EXEC` [logging channels](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/logging-overview#logging-channels) to an Azure Monitor log table prefixed with `sql`, and overrides (disables) the global redaction configuration for just these two log channels only.
+            - Sends log entries in the `OPS`, `HEALTH`, and `STORAGE` [logging channels](https://www.cockroachlabs.com/docs/{{site.current_cloud_version}}/logging-overview#logging-channels) to an Azure Monitor log table prefixed with `devops`, but only for those entries that are of log [severity level](/docs/{{site.current_cloud_version}}/logging.html#logging-levels-severities) `WARNING` or higher.
+            - Sends log entries in all other [logging channels](#what-log-channels-are-supported) to the Azure Monitor log table prefixed with `default`.
+
+        1. Once you have determined the configuration you'd like to use, edit the configuration to be a single line, the required form for passing to the configuration command in the next step. To accomplish this easily, use a third party minifier, such as [json minifier](https://jsonformatter.org/json-minify). The above configuration becomes the following single line, suitable for the next step's `POST` command:
```

Comment:
```suggestion
        1. Once you have determined the configuration you'd like to use, edit the configuration to be a single line, the required form for passing to the configuration command in the next step. To accomplish this, you can use a third-party minifier, such as [json minifier](https://jsonformatter.org/json-minify). The example configuration becomes the following single line, suitable for the next step's `POST` command:
```

You could also use "preceding configuration example". Best to avoid directional language for accessibility.